test/unit-test-helpers/mock_ima.js

```diff
@@ -64,7 +64,8 @@ google =
       setPlayerVersion : function() {},
       setPlayerType : function() {},
       setVpaidMode : function() {},
-      setLocale : function() {}
+      setLocale : function() {},
+      setDisableFlashAds : function() {},
```

Comment:
Can we add a unit test to check that the page level setting reaches this mock setDisableFlashAds function with the correct value?